### PR TITLE
docs: refine docs further

### PR DESCRIPTION
**Pull request by ankur-arch**
Update the readme to reflect recent changes in Railway and Pulse

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,22 +1,29 @@
-# Pulse starter project
+# Prisma Pulse starter project
 
 This repository has been created to help you get started with [Pulse](https://prisma.io/pulse). You will be able to use this project with any Pulse-ready PostgreSQL database. This project comes with a basic [`prisma.schema`](./prisma/schema.prisma) file as well as a Pulse subscription found in the [`index.ts`](./index.ts) file.
```
Comment (nikolasburk) [inline README.md:3]:
```suggestion
This repository has been created to help you get started with [Pulse](https://prisma.io/pulse). You will be able to use this project with any Pulse-ready PostgreSQL database. This project comes with a basic [`schema.prisma`](./prisma/schema.prisma) file as well as a Pulse subscription found in the [`index.ts`](./index.ts) file.
```

File: README.md [hunk 2]
```diff
@@ -1,22 +1,29 @@
...
+- [Local development](#local-development)
+  - [1. Clone the respository](#1-clone-the-respository)
+  - [2. Create and fill out a `.env` file](#2-create-and-fill-out-a-env-file)
+  - [3. Run the database migration](#3-run-the-database-migration)
+  - [4. Start the Pulse subscription](#4-start-the-pulse-subscription)
+  - [5. Test the subscription](#5-test-the-subscription)
+- [Deployment](#deployment)
+  - [Deploy the Pulse Starter Project on railway.app](#deploy-the-pulse-starter-project-on-railwayapp)
+- [More example Pulse projects](#more-example-pulse-projects)
+- [More information about Pulse](#more-information-about-pulse)
```
Comment (nikolasburk) [inline README.md:17]: I'd drop the TOC, the READMe isn't that long that it feels necessary.

```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,22 +1,29 @@
...
+
+## Prerequisites
+
+To successfully run the project, you will need the following:
+
+- A Prisma Pulse-compatible database
+  - Refer to our comprehensive guide [here](https://www.prisma.io/docs/pulse/database-setup) for setup instructions, or deploy a Pulse-ready database on Railway [![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/pulse-pg?referralCode=VQ09uv)
+- An active [Prisma Data Platform](https://pris.ly/pdp) account for obtaining a Prisma Pulse API Key
```
Comment (nikolasburk) [inline README.md:25]:
```suggestion
- The **connection string** of a Pulse-compatible database (if you don't have one yet, you can configure your database following the instructions in our [docs](https://www.prisma.io/docs/pulse/database-setup) or [use a Railway template](https://railway.app/template/pulse-pg?referralCode=VQ09uv))
- A **Pulse API key** which you can get by enabling a Pulse in a project in your [Prisma Data Platform](https://pris.ly/pdp) account
```

File: README.md [hunk 4]
```diff
@@ -1,22 +1,29 @@
...
+  
 ## Local development
```
Comment (nikolasburk) [inline README.md:27]:
```suggestion
## Getting started
```

File: README.md [hunk 5]
```diff
@@ -69,23 +76,23 @@ This will run a basic subscription on the `User` table. The code can be found in
...
+ }
 
-	for await (const event of subscription) {
-		console.log("just received an event:", event);
-	}
+ for await (const event of subscription) {
+  console.log("just received an event:", event);
+ }
 }
 ```
```
Comment (nikolasburk) [inline README.md:89]: indendation seems off here. let's use 2 spaces per indentation level, not one:

```ts
async function main() {
  const subscription = await prisma.user.subscribe();

  if (subscription instanceof Error) {
    throw subscription;
  }

  for await (const event of subscription) {
    console.log("just received an event:", event);
  }
}
```

File: README.md [hunk 6]
```diff
@@ -94,131 +101,32 @@ The following instructions use Prisma Studio to create a new record in the `User
...
-		"email": "test@prisma.io",
-		"name": "test"
-	}
+ "action": "create",
+ "created": {
+  "id": 1,
+  "email": "test@prisma.io",
+  "name": "test"
+ }
 }
```
Comment (nikolasburk) [inline README.md:110]: again, let's indent with two spaces per indentation level:

```json
{
  "action": "create",
  "created": {
    "id": 1,
    "email": "test@prisma.io",
    "name": "test"
  }
}
```

File: README.md [hunk 7]
```diff
@@ -94,131 +101,32 @@ The following instructions use Prisma Studio to create a new record in the `User
...
 ```
 
-## Deploy a Pulse Starter Project on railway.app
+## Deployment
+
+### Deploy the Pulse Starter Project on railway.app
```
Comment (nikolasburk) [inline README.md:116]:
```suggestion
You can also deploy this project on Railway by following the instructions in our [docs](https://www.prisma.io/docs/pulse/database-setup/railway#setup-with-template-prisma-pulse-db--app).
```

File: README.md [hunk 8]
```diff
@@ -94,131 +101,32 @@ The following instructions use Prisma Studio to create a new record in the `User
...
 
 [![Deploy on Railway](https://railway.app/button.svg)](https://railway.app/template/pulse-starter?referralCode=VQ09uv)
 
 This project is used together with the [prisma/pulse-railway-pg-config](https://github.com/prisma/pulse-railway-pg-config) repo to build a template on [railway.app](https://railway.app).
 
 [![Watch the setup template setup video](https://img.youtube.com/vi/0nt7CLDqYeY/0.jpg)](https://www.youtube.com/watch?v=0nt7CLDqYeY)
```
Comment (nikolasburk) [inline README.md:121]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -94,131 +101,32 @@ The following instructions use Prisma Studio to create a new record in the `User
...
--   [Pulse Railway Postgres Configuration Template](https://github.com/prisma/pulse-railway-pg-config)
+- [Prisma Pulse examples](https://pris.ly/pulse-examples)
 
 ## More information about Pulse
 
--   [Pulse documentation](https://pris.ly/pulse-docs)
--   [Pulse setup video](https://www.youtube.com/watch?v=Lvn05wM26zs)
--   [Pulse announcement blog post](https://www.prisma.io/blog/introducing-pulse-jtu4UPC8ujy4)
+- [Pulse documentation](https://pris.ly/pulse-docs)
+- [Pulse GA announcement blog post](https://pris.ly/gh/pulse-ga)
```
Comment (nikolasburk) [inline README.md:132]:
```
## Resources

- [Prisma Pulse examples](https://pris.ly/pulse-examples)
- [Pulse documentation](https://pris.ly/pulse-docs)
- [Pulse GA announcement blog post](https://pris.ly/gh/pulse-ga)

**Review by grpatel — COMMENTED** [review]
I have no additional suggestions! 🙂

**Review by nikolasburk — APPROVED** [review]
great work, Ankur! 👏 👏 👏

**Pull request merged**